packages/@react-stately/utils/test/useControlledState.test.js

```diff
@@ -81,9 +81,8 @@ describe('useControlledState tests', function () {
     let button = getByRole('button');
     getByTestId('5');
     expect(renderSpy).toBeCalledTimes(1);
-    actDOM(() =>
-      userEvent.click(button)
-    );
+    // eslint-disable-next-line test-act/act-events-test
```

Comment:
should this be added to the lint?
could we check to see if the local name of the import act has been changed?

Reply:
My rule actually only checks `Identifier === "act"` 😬  Should I change it so that it actually checks if it's the act from the testing library?

Reply:
yep, that's my thought, we might rename it to anything, like we do here

Reply:
Done